Source/WebCore/style/StyleExtractorSerializer.h

```diff
@@ -510,6 +510,12 @@ inline void ExtractorSerializer::serializeTextTransform(ExtractorState& state, S
     appendOption(TextTransform::FullWidth, CSSValueFullWidth);
     appendOption(TextTransform::FullSizeKana, CSSValueFullSizeKana);
 
+    if (textTransform.contains(TextTransform::MathAuto)) {
+        if (listEmpty)
+            serializationForCSS(builder, context, state.style, CSS::Keyword::MathAuto { });
+        listEmpty = !listEmpty;
+    }
```

Comment:
I'm not sure I understand this. From what you say above, the places where `TextTransform::MathAuto` is added is constrained by the `Source/WebCore/css/CSSProperties.json` syntax, where it cannot be used in combination with other values. So shouldn't we always have `listEmpty = true` here? And so the code could just be:

```
 if (textTransform.contains(TextTransform::MathAuto)) {
        serializationForCSS(builder, context, state.style, CSS::Keyword::MathAuto { });
        listEmpty = false;
 }
```

but maybe I'm missing something. If so, it would be good to have tests showing why this simple code cannot work.